packages/dts-generator/src/js-utils/ui5-metadata.js

```diff
@@ -25,13 +29,18 @@ const { log } = require("./logger");
  * @param {string} version
  * @returns {Promise<Record<string, UI5Lib>>}
  */
-async function getSAPUI5LibsMeta(version) {
+export async function getSAPUI5LibsMeta(version) {
+  if (!version || version.length < 5) {
```

Comment:
length < 5 ? wouldn't a regex be better? 

https://github.com/SAP/openui5/blob/master/src/sap.ui.core/src/sap/base/util/Version.js#L11

Reply:
done